01_process_occurrences.Rmd

```diff
@@ -44,69 +59,139 @@ Filter species of interest
++-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Yellow Warbler (YEWA)       | Species of interest for the City of Fort Collins Natural Areas program.                                                   |
++-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Clouded Sulphur             | Positively correlated with detections of non-resident butterfly species.                                                  |
+|                             |                                                                                                                           |
+|                             | The presence of this species may indicate that the community and habitat can support a non-resident butterfly species.    |
++-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Orange Sulphur              | Positively correlated with butterfly species richness.                                                                    |
+|                             |                                                                                                                           |
+|                             | The presence of this species may indicate that the community and habitat can support a wide variety of butterfly species. |
++-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
+| Painted Lady                | Positively correlated with butterfly species richness.                                                                    |
+|                             |                                                                                                                           |
+|                             | The presence of this species may indicate that the community and habitat can support a wide variety of butterfly species. |
++-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
 
 ```{r}
 # double-check all species are spelled correctly
-butterfly_obs %>% distinct(species) %>% View()
+butterflies %>% distinct(scientific_name) %>% View()
 
-bird_obs %>% distinct(species) %>% View()
-# WEME has two distinct obs, likely an extra species. Remove spacies in all species name entires below
+birds %>% distinct(species) %>% View()
+# WEME has two distinct obs, likely an extra species. Remove spaces in all species name entires below
 
 
-obs_final <- bird_obs %>% 
+obs_final <- birds %>% 
```

Comment:
Are we differentiating between volunteer vs. technician obs? I'm guessing not, and it's probably fine but just curious + something we should note in reports/publications.